scrapers_next/pa/committees.py

```diff
@@ -0,0 +1,171 @@
+from spatula import CSS, HtmlPage, HtmlListPage, SelectorError
+from openstates.models import ScrapeCommittee
+
+
+class HouseCommitteeDetail(HtmlPage):
+    example_source = "https://www.legis.state.pa.us/cfdocs/CteeInfo/index.cfm?Code=32&CteeBody=H&SessYear=2021"
+    example_name = "Aging & Older Adult Services"
+    example_input = ScrapeCommittee(
+        name=example_name, classification="committee", chamber="lower"
+    )
+
+    def process_page(self):
+        com = self.input
+        try:
+            # This section has the chair memebers the regular, democratic and minority and the roles
+            # main chair
+            Chair_Member = (
```

Comment:
Still seeing a lot of variables with capital letters in them. I need you to follow [python style guide](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names), as we talked about previously. That means variable names are "lowercase, with words separated by underscores as necessary to improve readability".